API/Controllers/BooksController.cs

```diff
@@ -0,0 +1,104 @@
+﻿using Microsoft.AspNetCore.Authentication.JwtBearer;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using VL.Shared.Interfaces;
+using VL.Shared.Model;
+
+namespace API.Controllers
+{
+    [Authorize(AuthenticationSchemes = JwtBearerDefaults.AuthenticationScheme)]
+    [Route("api/vl")]
+    [ApiController]
+    public class BooksController : ControllerBase
+    {
+        private readonly IBookService _bookService;
+
+        public BooksController(IBookService bookService)
+        {
+            _bookService = bookService;
+        }
+
+
+        /// <summary>
+        /// GET: api/Books
+        /// </summary>
+        /// <returns></returns>
+        [HttpGet, Authorize("BasicUser")]
```

Comment:
In the context of your system "BasicUser" does not provide any insight into what policy you've created here - check out the policy naming conventions for Microsoft